content/innersource/codeowners-configuration.md

```diff
@@ -5,8 +5,9 @@ description: ""
 innersource: true
 hideToc: true
 ---
+For Github Code Owners is built in. See [About code owners](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) in the Github documentation for configuration details.
```

Comment:
I know this is a Draft and not ready for review yet, but make sure to use the GitHub capitalization (capital H) everywhere. :-)

Reply:
Done